### DISCUSSION TITLE
[2024-06-11] Incident Thread

### DISCUSSION THREAD

**Discussion by github-staff** [community/community#128173] [Actions] [answered]
We are investigating reports of degraded performance for Actions and API Requests.

_Subscribe to this Discussion for updates on this incident. Please upvote or emoji react instead of commenting +1 on the Discussion to avoid overwhelming the thread. Any account guidance specific to this incident will be shared in thread and on the [Incident Status Page](https://www.githubstatus.com/incidents/lfrlwdg67fn8)._

**Comment by github-staff** [comment]
## Update

Customers may see issues running Actions

**Comment by github-staff** [comment]
## Update

Customers may see issues running Actions, we are in the process of applying a mitigation to restore our service.

**Comment by github-staff** [comment]
## Update

We've applied a mitigation to unblock running Actions and are seeing an improvement in our service availability.

**Comment by github-staff** [comment]
## Resolved

This incident has been resolved.

**Comment by nturnerlivanova** [comment]
Are you sure? Self hosted runners are not picking up jobs

**Reply by seantleonard** [reply]
i had to cancel a job and re-run it. Ran quickly after that.

**Reply by nturnerlivanova** [reply]
Working

**Reply by Gareth-Nicholas-pay4later** [reply]
Actions failing, auth github.com

**Comment by github-staff** [comment, marked as answer]
## Summary


On June 11th, 2024 between 20:13 UTC and 21:39 UTC, the GitHub Actions service was degraded. A security-related change applied by one of our third-party providers prevented new customers from onboarding to GitHub Actions and caused an average 28% of Actions jobs to fail.

We mitigated the incident by working with the third-party provider to revert the change and are working with their engineering team to fully understand the root cause. Additionally, we are improving communication between GitHub and our service providers to reduce the time needed to resolve similar issues in the future.

**Comment by zualafqarmuhamad** [comment]
My tiktok account group

**Comment by bomba466** [comment]
i have been hacked they took certificates over

**Comment by bomba466** [comment]
don't have any files on the pc but on my cell
